pages/code-of-conduct.js

```diff
@@ -0,0 +1,23 @@
+import Link from "next/link";
+
+//import Footer from "../components/footer"
+
+export default function CodeOfConduct() {
```

Comment:
I even think you could erase this file, as it is not within this scope, and is unfinished.

Reply:
Yeah! Already